src/pages/about.jsx

```diff
@@ -0,0 +1,80 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { css } from 'react-emotion';
+import { mediaQueries } from '../utils/media-queries';
+import Container from '../components/Container';
+
+const AboutPage = ({ data }) => (
+  <div>
+    <Container
+      className={css`
+        margin-top: 5rem;
```

Comment:
Instead of using a magic constant which should be equal on every page, I would recommend only enabling the `position: absolute` CSS rule of the index layout's `Headroom` component if `isHomepage` is true. _(See https://github.com/simonyiszk/konferencia-web/pull/9/files#diff-206bf21048ba13c26535322e286ab844R131 for reference.)_